aiida/manage/configuration/setup.py

```diff
@@ -210,26 +210,23 @@ def delete_db(profile, non_interactive=True):
     """
     from aiida.manage import get_config
     from aiida.manage.external.postgres import Postgres
-    from aiida.common import json
 
-    pconfig = profile.dictionary
+    pdict = profile.dictionary
 
-    postgres = Postgres(port=pconfig.get('AIIDADB_PORT'), interactive=not non_interactive, quiet=False)
-    postgres.dbinfo['user'] = pconfig.get('AIIDADB_USER')
-    postgres.dbinfo['host'] = pconfig.get('AIIDADB_HOST')
+    postgres = Postgres.from_profile(profile, interactive=not non_interactive, quiet=False)
     postgres.determine_setup()
 
-    echo.echo(json.dumps(postgres.dbinfo, indent=4))
+    # echo.echo(json.dumps(postgres.get_dbinfo(), indent=4))
```

Comment:
delete commented line

Reply:
I wasn't so sure what to do with this.
As it stands it would be printing the way you connect for deleting the DB (e.g. using database "template1" instead of the actual database name). 
My feeling was that this was not very helpful information without providing some context.

Reply:
I am not too familiar with this part of the code, I have never really used/needed it myself. But I agree, as it stands, without context, it might not be too useful. I would be fine with either adding a bit more information or deleting it for the time being